elements/urth-viz-chart/urth-viz-chart-common-behavior.html

```diff
@@ -162,6 +156,16 @@
                 },
 
                 /**
+                 * UI information based on the current selection
+                 */
+                selectionInfo: {
+                    type: Array,
+                    value: function () { return []; },
+                    readOnly: true,
+                    notify: true
+                },
+
+                /**
                  * Allows user to select multiple chart components (bar in barchart, slice in pie chart, etc).
                  */
                 multiSelect: {
```

Comment:
Only because urth-viz-table does not yet support it. I made a note of this in the issue requesting that feature